charts/sn-platform/values.yaml

```diff
@@ -1966,6 +1966,8 @@ streamnative_console:
     GOOGLE_REDIRECT_URI: ""
     GRAFANA_AUTH_PROXY: false
     GRAFANA_AUTH_PROXY_USER: ""
+    CONNECTOR_ENABLED: false
+    FLINK_SQL_ENABLED: false
```

Comment:
I don't think we should add `FLINK_SQL_ENABLED` on console now, cause we don't provide Flink Cluster on sn-platform now.

Reply:
I removed the flink sql configuration `FLINK_SQL_ENABLED` from the value file, but kept the environment variables in statefulset, what do you think?